islandora-deprecated/drupal-project
# Drupal 8.7

Ticket: https://github.com/Islandora-CLAW/CLAW/issues/1110

This is the first time I am doing this, so please do review it carefully.

## What's new?
* merged it with upstream
* `sudo composer update` in my own machine

## How should this be tested?
* Cone the latest claw-playbook
* Edit [this line](https://github.com/Islandora-Devops/claw-playbook/blob/master/inventory/vagrant/group_vars/webserver/drupal.yml#L20) to point to this branch (`drupal_composer_project_package: "islandora/drupal-project:dev-drupal_87"`)
* vagrant up
* verify that the red error message we used to get no longer shows!
* Verify Drupl 8.7 is installed - http://localhost:8000/admin/reports/status
* Do basic Islandora sanity checks - (i.e create new objects and verify derivatives are created, synched properly)


@Islandora-CLAW/committers
